lgtm.yml

```diff
@@ -0,0 +1,6 @@
+extraction:
+  java:
+    index:
+      properties_files: true
+      build_command:
+        - ./gradlew -Dorg.gradle.jvmargs=-Xmx2g --no-daemon -S build
```

Comment:
I believe the task that we want is `assemble` or `compile`. `build` tries to run tests, which won't work headlessly.

Reply:
Resolved! Checks work again.